woocommerce/woocommerce
# Remove unused large banner image

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

This removes an unused image of 5.7mb in size that wasn't used anymore.

### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Load this branch and run `pnpm run build` everything should build correctly
2. Create a new WooCommerce installation with this version. 
3. Install  WooCommerce Block [Nightly](https://github.com/woocommerce/woocommerce-blocks/releases/tag/nightly)
4. Make sure to enable `customize-store` feature flag
5. Navigate to the CYS task ( it should look correctly ).
6. Complete the CYS flow 
7. Observe that if you revisit the completed task everything looks correctly ( there shouldn't be a missing image in the banner ).
8. Run `pnpm --filter=woocommerce run build:zip` and unpack the generated zip file
9. Check the `woocommerce/assets/client/admin` folder and make sure there is no 5.7mb svg file



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
